scripts/README.md

```diff
@@ -177,4 +178,13 @@ The output is following format.
   {2} : The column number that a control-code exists.
   {3} : The found control-code.
   {4} : The one-line strings in the file.
-```
\ No newline at end of file
+```
+
+## ja/verify-spelling.sh
+
+This script finds invalid Japanese words what are against the guideline[1]
```

Comment:
maybe sometimes they're not "invalid" though

suggest:
```suggestion
This script finds Japanese words that are against the guideline[1]
```

Reply:
Thanks for your review @nasa9084 
That makes sense, I applied the suggestion on the latest pr.